client/client.go

```diff
@@ -76,6 +77,7 @@ func GetRawQuote(quoteProvider any, reportData [64]byte) ([]uint8, error) {
 // getRawQuoteViaDevice uses TDX device driver to call getReport for report and convert it to
 // quote using an ioctl call.
 func getRawQuoteViaDevice(d Device, reportData [64]byte) ([]uint8, error) {
+	logger.V(1).Info("Get raw quote via Device")
```

Comment:
By default, verbosity is set to 0. So added current INFO logs at verbosity 1 so that cmdline flag could be used to enable to disable them.

Reply:
Changing the log to "Get raw TDX quote via <device path> Device" would make it more readable.

Reply:
Done.